### PR TITLE
Readded reverted changes for additional review.

### PR DESCRIPTION
**Pull request by JReissmueller**

### CODE REVIEW

**Comment by JReissmueller** [comment]
I messed this one up by merging when there was an outstanding comment from the KH folks.  So I reverted the merge then readded the changes and made a new PR here.  I'm copying the comment below. From @nicholsk18

Hey @abdyfranco  I see that you added US and CA but I think intlNumber() still returns the country dialing code which I think needs to be stripped for this as well.

From namesilo documentation Country dialing code SHOULD NOT be passed for any countries.
Feel free to ignore this if you adjusted intlNumber() function but just wanted to mention that!

Thanks again for fixing this!

**Comment by abdyfranco** [comment]
> I messed this one up by merging when there was an outstanding comment from the KH folks. So I reverted the merge then readded the changes and made a new PR here. I'm copying the comment below. From @nicholsk18
> 
> ```
> Hey @abdyfranco  I see that you added US and CA but I think intlNumber() still returns the country dialing code which I think needs to be stripped for this as well.
> 
> From namesilo documentation Country dialing code SHOULD NOT be passed for any countries.
> Feel free to ignore this if you adjusted intlNumber() function but just wanted to mention that!
> 
> Thanks again for fixing this!
> ```

Updated

File: namesilo.php [hunk 1]
```diff
@@ -2891,7 +2897,17 @@ private function formatPhone($number, $country)
             Loader::loadModels($this, ['Contacts']);
         }
 
-        return $this->Contacts->intlNumber($number, $country, '.');
+        $formatted_phone = preg_replace('/[^0-9]+/', '', $number ?? '');
```
Comment (JReissmueller) [inline namesilo.php:2900]: hmm this doesn't do anything to remove country code for places other than CA and the US.  I think one slightly more robust option would be to use intlNumber() and then exploding on the . separator and grabbing the last peice. You could still limit US and CA numbers to 10 digits

Comment (abdyfranco) [reply namesilo.php]: Updated

File: namesilo.php [hunk 2]
```diff
@@ -2891,7 +2897,19 @@ private function formatPhone($number, $country)
             Loader::loadModels($this, ['Contacts']);
         }
 
-        return $this->Contacts->intlNumber($number, $country, '.');
+        $phone = $this->Contacts->intlNumber($number, $country, '.');
+        $phone_parts = explode('.', $phone, 2);
+        $formatted_phone = preg_replace('/[^0-9]+/', '', $phone_parts[1] ?? '1111111111');
```
Comment (JReissmueller) [inline namesilo.php:2902]: I think $phone would make a better default

File: namesilo.php [hunk 3]
```diff
@@ -2891,7 +2897,19 @@ private function formatPhone($number, $country)
...
+
+        if (in_array($country, ['US', 'CA'])) {
+            $formatted_phone = substr($formatted_phone, -10);
+
+            if (strlen($formatted_phone) !== 10) {
+                $formatted_phone = '1111111111';
```
[the remark this answers is not in the document]
Comment (abdyfranco) [reply namesilo.php]: Updated

**Pull request merged**